### DISCUSSION TITLE
ckan install from source failure

### DISCUSSION THREAD

**Discussion by turellaa** [ckan/ckan#6647] [Q&A]
starting from the page : "https://github.com/ckan/ckan/wiki/How-to-Install-CKAN-2.9-on-CentOS-7"
we have run the follow command : 
--
pip install -e 'git+https://github.com/ckan/ckan.git@ckan-2.9.0#egg=ckan[requirements]'
--
and received 
--
ckan from git+https://github.com/ckan/ckan.git@ckan-2.9.0#egg=ckan does not appear to be a Python project: neither 'setup.py' nor 'pyproject.toml' found 
--
can someone help me to understand why ?

**Comment by jqnatividad** [comment]
Have you tried running with `pip3` instead?

`pip3 install -e 'git+https://github.com/ckan/ckan.git@ckan-2.9.0#egg=ckan[requirements]'`

And if that's the solution, can you edit the wiki page accordingly (and maybe your post above as well as its in **ALL CAPS bold?**)
